core/input/input_event.cpp

```diff
@@ -805,9 +814,9 @@ String InputEventMouseMotion::to_string() {
 			break;
 	}
 
-	// Work around the fact vformat can only take 5 substitutions but 6 need to be passed.
-	String mask_and_position = vformat("button_mask=%s, position=(%s)", button_mask_string, String(get_position()));
-	return vformat("InputEventMouseMotion: %s, relative=(%s), velocity=(%s), pressure=%.2f, tilt=(%s)", mask_and_position, String(get_relative()), String(get_velocity()), get_pressure(), String(get_tilt()));
+	// Work around the fact vformat can only take 5 substitutions but 7 need to be passed.
+	String mask_and_position_and_relative = vformat("button_mask=%s, position=(%s), relative=(%s)", button_mask_string, String(get_position()), String(get_relative()));
+	return vformat("InputEventMouseMotion: %s, velocity=(%s), pressure=%.2f, tilt=(%s), pen_inverted=(%d)", mask_and_position_and_relative, String(get_velocity()), get_pressure(), String(get_tilt()), get_pen_inverted());
```

Comment:
```suggestion
	return vformat("InputEventMouseMotion: %s, velocity=(%s), pressure=%.2f, tilt=(%s)%s", mask_and_position_and_relative, String(get_velocity()), get_pressure(), String(get_tilt()), get_pen_inverted() ? ", pen_inverted" : "");
```

> This is getting too lengthy. Any better ideas?

It's for debug purpose only, so should be fine as is.